### PR TITLE
feat: add aliases for name of arguments

### LINKED ISSUE #150: Parsing of groupped options
I've found next case in options section:
`--color, --no-color            Force enabling/disabling of color`

It that case 2 errors occurs:
- unnecessary prefix "args" - `isFlag:false`
- `--no-color` wasn't parsed

Please, fix it.

### PR DESCRIPTION
**Pull request by ifedchankau**
- Change type of argument alias field [string if 1 alias, array if 2 or more]
- Add parsing of argument alias [if argument have several long / short names]
- Add tests for argument alias

Closes #150

### CODE REVIEW

File: src/arguments.js [hunk 1]
```diff
@@ -7,23 +10,39 @@
  * @return {Object} - argument template
  */
 const getArgumentObject = (object, context) => {
-    let argument = context.get.template.argument();
-    argument = {
+    let argument = {
         enum: getEnum(object.arg + object.description, context),
-        longName: getPropertyName(object.arg, context.regexp.argument.long),
-        shortName: getPropertyName(object.arg, context.regexp.argument.short),
+        longNames: getPropertyNames(object.arg, context.regexp.argument.long),
```
Comment (romanrostislavovich) [inline src/arguments.js:15]: I think we don't need to save this array of names to `longNames` or `shortNames`

File: src/arguments.js [hunk 2]
```diff
@@ -7,23 +10,39 @@
...
+/**
+ * Get alias from names of an argument
+ * @param {object} argument - argument object with description, names, etc.
+ * @return {Array} alias - alias of argument
+ */
+const getAlias = (argument) => {
+    let alias = [];
+    let longNames = argument.longNames;
+    let shortNames = argument.shortNames;
+    if (longNames && longNames[1]) alias.push(longNames.slice(1));
```
Comment (romanrostislavovich) [inline src/arguments.js]: Many `if` in this function, it's need to improve

File: src/arguments.js [hunk 3]
```diff
@@ -181,8 +208,9 @@ const checkFlagsByExamples = (section, context) => {
  * @return {string} - result string
  */
 const removeExtraArgumentNames = (string, argument) => {
-    return removeChar(string,
-        [',', argument.longName, argument.shortName]).trim();
+    let remove = [','].concat(argument.longNames).concat(argument.shortNames);
```
Comment (romanrostislavovich) [inline src/arguments.js]: Mb better use `[',', argument.longNames, .. ]` ?

File: src/arguments.js (hunk 1 above)
Comment (ifedchankau) [reply src/arguments.js:15]: @svoboda-rabstvo it is using in 3 functions later. I think that it is more convenient to send one object `argument` than 3 elements: `object argument`, `array longNames`, `array shortNames`.

File: src/arguments.js (hunk 2 above)
Comment (ifedchankau) [reply src/arguments.js]: @svoboda-rabstvo yes, but it is really needed there. Maybe, you have other solution?

File: src/arguments.js (hunk 3 above)
Comment (ifedchankau) [reply src/arguments.js]: @svoboda-rabstvo no, in that case array will be included in other array. 

For example:
``` 
longNames = ['--color', '--no-color'];
shortNames = ['-c', '-n'];
```
remove array will be:
```
remove = [',', ['--color', '--no-color'], ['-c', '-n']];
```

**Review by romanrostislavovich — APPROVED** [review]
+1

**Review by romanrostislavovich — APPROVED** [review]
+1

**Pull request merged**